internal/manifests/cronjob/build.go

```diff
@@ -0,0 +1,90 @@
+package cronjob
+
+import (
+	batchv1 "k8s.io/api/batch/v1"
+	batchv1beta1 "k8s.io/api/batch/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Builder represents the type to build Cronjob objects
+type Builder struct {
```

Comment:
similar ask as above about putting this into another file

Reply:
Same as [reply](https://github.com/openshift/elasticsearch-operator/pull/735#discussion_r655862449)